RecoVertex/Configuration/python/RecoVertex_cff.py

```diff
@@ -82,3 +84,4 @@
 phase2_timing_layer.toReplaceWith(offlinePrimaryVertices4DWithBS, offlinePrimaryVertices4DwithPIDWithBS.clone())
 phase2_timing_layer.toModify(offlinePrimaryVertices4D, vertices = "unsortedOfflinePrimaryVertices4D", particles = "trackRefsForJetsBeforeSorting4D")
 phase2_timing_layer.toModify(offlinePrimaryVertices4DWithBS, vertices = "unsortedOfflinePrimaryVertices4D:WithBS", particles = "trackRefsForJetsBeforeSorting4D")
+phase2_timing_layer.toModify(tofPID, vtxsSrc='unsortedOfflinePrimaryVertices4D')
```

Comment:
normally only modules defined in a given file can have toModify applied.
Please put the ```phase2_timing_layer.toModify(tofPID``` in RecoVertex_phase2_timing_cff.py